Core/include/Acts/Vertexing/GaussianTrackDensity.hpp

```diff
@@ -146,7 +154,7 @@ class GaussianTrackDensity {
   ///
   /// @return Track density, first and second derivatives
   std::tuple<double, double, double> trackDensityAndDerivatives(State& state,
-                                                                double z) const;
+                                                                double z,double time) const;
```

Comment:
I think we should switch to an `Eigen` type here but not crucial. Apart from that we will need a matrix or a bigger array to store the new derivatives. Right now we have `density, d(density)/dz, d^2(density)/dz^2` but we will also need the `t` terms and the mixed term with `dzdt`